examples/mock-identity/mockoon-bbidentity.json

```diff
@@ -620,6 +620,212 @@
       "enabled": true,
       "responseMode": null,
       "type": "http"
+    },
+    {
+      "uuid": "4370f082-8742-4e17-8a05-0a5643f69e1e",
+      "type": "http",
+      "documentation": "Wallet binding endpoint is invoked by Wallet's backend server.",
+      "method": "post",
+      "endpoint": "wallet-binding",
+      "responses": [
+        {
+          "uuid": "9d3cfad3-8697-4f8d-b685-88f06f12f1cc",
+          "body": "{\n  \"responseTime\": \"string\",\n  \"response\": {\n    \"walletUserId\": \"string\",\n    \"certificate\": \"string\",\n    \"expireDateTime\": \"string\"\n  },\n  \"errors\": []\n}",
+          "latency": 0,
+          "statusCode": 200,
+          "label": "Successfully generated wallet user id",
+          "headers": [
+            {
+              "key": "Content-Type",
+              "value": "application/json"
+            }
+          ],
+          "bodyType": "INLINE",
+          "filePath": "",
+          "databucketID": "",
+          "sendFileAsBody": false,
+          "rules": [
+            {
+              "target": "body",
+              "modifier": "requestTime",
+              "value": "",
+              "invert": true,
+              "operator": "null"
+            },
+            {
+              "target": "body",
+              "modifier": "request.individualId",
+              "value": "^[a-zA-Z0-9]+$",
+              "invert": false,
+              "operator": "regex"
+            },
+            {
+              "target": "body",
+              "modifier": "request.authFactorType",
+              "value": "^[a-zA-Z0-9]+$",
+              "invert": false,
+              "operator": "regex"
+            },
+            {
+              "target": "body",
+              "modifier": "request.format",
+              "value": "^[A-Za-z0-9]+-[A-Za-z0-9]+$",
+              "invert": false,
+              "operator": "regex"
+            },
+            {
+              "target": "body",
+              "modifier": "request.challengeList",
+              "value": "",
+              "invert": true,
+              "operator": "empty_array"
+            },
+            {
+              "target": "body",
+              "modifier": "request.publicKey",
+              "value": "^([a-zA-Z0-9_=]+)([a-zA-Z0-9_=]+)([a-zA-Z0-9_\\\\-\\\\+\\\\/=]*)",
```

Comment:
Is it working? When I paste this regex to the regex validator it's not match with the publicKey...
"^([a-zA-Z0-9_=]+)([a-zA-Z0-9_=]+)([a-zA-Z0-9_\\-\\+\\/=]*)" - paste it to the mockoon-bbidentity.json, not to the Mockoon app (here should be `^([a-zA-Z0-9_=]+)([a-zA-Z0-9_=]+)([a-zA-Z0-9_\\-\\+\\=]*)`)

Reply:
updated